docs/docsite/rst/index.rst

```diff
@@ -47,16 +47,10 @@ Ansible releases a new major release of Ansible approximately three to four time
 
 .. toctree::
    :glob:
-   :maxdepth: 2
-   :caption: Scenario Guides
-
-   scenario_guides/guide_*
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Ansible for VMWare
+   :maxdepth: 1
+   :caption: Integrating Ansible with Other Tools
```

Comment:
I read this as **Modifying Ansible to work together with other tools** which is not what this content is about.

These *Scenario Guides* describe for different technologies (or infrastructure) how to best automate or manage that infrastructure/product.

"Tools" here is also a misnomer IMO, Ansible is a tool, but your production database is not a tool.

> **integrate**  *verb*
>   combine (one thing) with another to form a whole.
>   *"transport planning should be integrated with energy policy"*
>   *synonyms:*  combine, amalgamate, merge, unite, join, fuse, blend, mingle, coalesce, consolidate, meld, intermingle, mix, intermix, incorporate, affiliate, unify, assimilate, homogenize, harmonize, mesh, desegregate; literarycommingle
>    *"he proposes to integrate our reserve forces more closely with the regular forces"*

Not sure what I have to do to get this point across...

Reply:
@dagwieers it's still a WIP, and your point is well taken. I searched for a term that describes all the technologies covered in this section of the docs. The word "tool" is not my favorite, but was the best catch-all term I had come up with so far.

The Scenario Guides are too numerous to appear in the left-nav. Right now with `Scenario Guides` as the caption and the various Guide titles below, the user can get the idea pretty easily.
If we collapse the TOC and keep the name Scenario Guides, what can we use as the Caption above it? The other sections start a pattern:
`Using Ansible: User Guide`
and
`Contributing to Ansible: Community Guide`
So what would work for the Scenario Guides section?
`Managing Technology with Ansible: Scenario Guides`?

Reply:
If the Scenario Guides cannot be at the left in full, that's fine, but I don't see what changing the name fixes. Unless we have a better name that is.

**Scenario Guides**
*Automate and manage other technologies with Ansible*

Although I think I like **Automation Guides** better now, but that's just a personal preference.

Reply:
I think we can leave `automation` out anyway - if it's Ansible, it's automation by definition.